usecase/usecase-webapi-xwalk-tests/samples/WebCL/res/COPYING

```diff
@@ -0,0 +1,5 @@
+index.html come from
+https://github.com/uebquite/webcl-demos
+
+These tests are copyright by the Khronos Group under MIT License:
+https://www.khronos.org/registry/webgl/sdk/tests/test-guidelines.md
```

Comment:
However, I cannot agree to this license for https://github.com/uebquite/webcl-demos/blob/master/Hello/index.html

They are totally 2 different projects! And I find no license from https://github.com/uebquite/webcl-demos